zone/guild_mgr.cpp

```
@@ -651,7 +674,111 @@ void ZoneGuildManager::ProcessWorldPacket(ServerPacket *pack)
+				link_data[0] == '\0' ||
+				link_data[link_data_size - 1] != '\0' ||
+				link_data[link_data_size - 2] != '^' ||
+				std::memchr(link_data, '\0', link_data_size - 1) != nullptr ||
+				!is_zone_loaded
+			) {
+				return;
+			}
+
+			const auto update_size =
+				static_cast<uint32>(sizeof(RoF2GuildAchievementUpdateHeader) + link_data_size);
+			EQApplicationPacket outapp(OP_GuildUpdate, update_size);
```

Clang (14.0.6)

[warning] variable 'outapp' is not initialized

(cppcoreguidelines-init-variables)